vllm/model_executor/models/mlp_speculator.py

```diff
@@ -32,18 +34,26 @@ def __init__(
         self,
         normalized_shape,
         eps=1e-06,
+        elementwise_scale=True,
+        elementwise_shift=False,
```

Comment:
Curious why the True/False default values. Looks like these are only used as both False or both True..

Reply:
I don't see why we need two variables actually, I have changed it accordingly.